src/Compute/Compute.Autorest/custom/Set-AzVMRunCommand_ScriptLocalPath.ps1

```diff
@@ -0,0 +1,190 @@
+
+# ----------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Code generated by Microsoft (R) AutoRest Code Generator.Changes may cause incorrect behavior and will be lost if the code
+# is regenerated.
+# ----------------------------------------------------------------------------------
+
+function Set-AzVMRunCommand_ScriptLocalPath {
+    [OutputType([Microsoft.Azure.PowerShell.Cmdlets.Compute.Models.Api20210701.IVirtualMachineRunCommand])]
+    [CmdletBinding(PositionalBinding=$false, SupportsShouldProcess, ConfirmImpact='Medium')]
```

Comment:
Is the DefaultParameterSetName missing here?

Reply:
do i need to set that? looking at other customized cmdlets in the folder, it doesn't seem necessary. Looking at the doc, it looks like it resolves the original as the defaultParameterSet by default. 
https://github.com/Azure/azure-powershell/pull/19935/files#diff-98b2e209fe660fbafe42a698f08f3105e161521fdca07891934035f26f317b01R15